Gemfile

```diff
@@ -1,4 +1,26 @@
 source 'https://rubygems.org'
 
-gem 'cocoapods', '1.4.0'
-gem 'xcpretty'
+gem 'cocoapods', '1.5.0'
+
+# xcpretty still does not support testing multiple devices in parallel
+gem 'xcpretty', :git => 'https://github.com/Raizlabs/xcpretty.git', :branch => 'feature/parallel-testing-support'
+
+# Danger
+group :test, :danger do
```

Comment:
You don't need the test and danger groups when developing on your machine, only on CI. Maybe there's a better way of doing this?

Reply:
I didn't see anywhere where the test and danger groups are actually used, but maybe I wasn't looking in the right place, since I don't know much about Gemfiles.

Reply:
Yeah me neither, I've just seen it before. I think right now it just installs everything in all groups and I'd need to configure it differently to not auto-install those groups.